homeassistant/components/fritzbox/coordinator.py

```diff
@@ -87,6 +87,22 @@ def _update_fritz_devices(self) -> FritzboxCoordinatorData:
             for template in templates:
                 template_data[template.ain] = template
 
+        if self.data:
+            new_devices = [ain for ain in device_data if ain not in self.data.devices]
+            if new_devices:
+                self.hass.bus.fire(
```

Comment:
Don't use the event bus for intra integration communication. We have the dispatcher helper for that purpose.